community/kernel/src/main/java/org/neo4j/kernel/impl/store/id/IdGeneratorImpl.java

```diff
@@ -267,6 +267,11 @@ public static long readHighId( FileSystemAbstraction fileSystem, File file ) thr
         return IdContainer.readHighId( fileSystem, file );
     }
 
+    public static long readDefragCount( FileSystemAbstraction fileSystem, File file ) throws IOException
```

Comment:
I would like to see some javadoc specifying the limitations here, i.e. it's not safe to run when the `FreeIdKeeper` is active since it has internal buffers not accounted for.